server/src/sass/components/map/_ol.scss

```diff
@@ -276,4 +276,4 @@
 .ol-overviewmap .ol-overviewmap-box:hover {
     cursor: move;
 }
-*/
+*/
```

Comment:
I know you haven't done anything to this file either, but there is also a huge amount of commented out lines, are they needed?